libs/designer-ui/src/lib/settings/settingsection/settingTokenField.tsx

```diff
@@ -60,9 +61,20 @@ export interface SettingTokenFieldProps extends SettingProps {
 }
 
 export const SettingTokenField = ({ ...props }: SettingTokenFieldProps) => {
+  const intl = useIntl();
```

Comment:
will we want this to show in all cases including the simplequerybuilder or hybridquerybuilder?

Reply:
The intention is to show the description for all Condition actions.

Are simplequerybuilder and hybridquerybuilder also a type of Condition action? If so, can you share 
1) how they can be identified in code
2) how can I verify the UI for them?

Reply:
yes they are also of condition editor type, they are diffrentiated with editorOption: "isOldFormat" for simple query builder, and "isHybridEditor" for hybrid query builder. If you are doing this change in the settingtokenfield, it can be accessed with the editorViewModel.isOldFormat; however, if this is only wanted on the condition editor, then we can just move this description to the ui component where condition editor is hosted, and give it an optional prop to show that description: libs\designer-ui\src\lib\querybuilder\index.tsx

Reply:
Moved the description to querybuilder file.